implementations/index.md

```diff
@@ -339,6 +339,7 @@ includes related resources.
 
 * [Endpoints provides a fully working example API](http://github.com/endpoints/example/)
 * [Sinja provides a fully-working example API](https://github.com/mwpastore/sinja/tree/master/demo-app)
+* [Hatchify provides a full-working example API and client](https://stackblitz.com/edit/bitovi-hatchify-t7amou?embed=1&file=backend%2Findex.ts)
```

Comment:
I fear this section explodes if we start adding example projects hosted on Stackblitz.

```suggestion
```